relay/circuit-v1.md

```diff
@@ -0,0 +1,294 @@
+# Circuit Relay v0.1.0
+
+> Circuit Switching for libp2p, also known as TURN or Relay in Networking literature.
```

Comment:
Nit pick: I don't think this needs to be duplicated once more, given it already being mentioned in the `README.md`.
```suggestion
```